### PR TITLE
Create v001_manual_acceptance_tests.md

### PR DESCRIPTION
**Pull request by NuuttiSten**
Tests to validate v0.0.1 release

### CODE REVIEW

File: test/v001_manual_acceptance_tests.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Helsinki ML Project Template, Release v0.0.1, Acceptance tests
+
+The purpose of these tests is to validate the usability of the template to the end users.
+The tests focus on the template instructions - can the user with no prior experience with the template understand it, install it and begin work based on the instructions?
+
+After you have carried out a test, add PASS / FAIL + YOUR NAME + DATE after the test. If the test FAILS, make an issue where you describe the problem in detail. Add the issue number to the list after the test. Unclear instructions are a valid reason for an issue.
+
+## 1. Review documentation on GitHub: https://github.com/Datahel/ml_project_template/tree/master
+1.1 Read through README.md. Check for broken formatting, typos and mark down if there are any parts you don't understand. Check that all links in README work, i.e. lead to a sensible page or hyperlink to some part of the template. Acceptance criteria: nothing to fix (format breakdowns may have to be accepted because README / html conversions).
```
Comment (jussiarpalahti) [inline test/v001_manual_acceptance_tests.md:9]: It seems Github markdown doesn't get the numbered list from this format. Text just flows together.

**Comment by jussiarpalahti** [comment]
LGTM (except the Markdown formatting)

**Review by jussiarpalahti — CHANGES_REQUESTED** [review]

File: test/v001_manual_acceptance_tests.md (hunk 1 above)
Comment (NuuttiSten) [reply test/v001_manual_acceptance_tests.md:9]: Whoops, need to fix this. I think it requires two newlines to understand the list

Comment (NuuttiSten) [reply test/v001_manual_acceptance_tests.md:10]: Ok, it still does not understand this as list. MD has limitations on lists, if the format breaks down in the middle. I suggest we go with this.

Comment (NuuttiSten) [reply test/v001_manual_acceptance_tests.md]: https://meta.stackexchange.com/questions/85474/how-to-write-nested-numbered-lists

Comment (jussiarpalahti) [reply test/v001_manual_acceptance_tests.md]: Yes, combining headings with sublists isn't possible in Markdown. I think we can live without the styling.

**Review by jussiarpalahti — APPROVED** [review]
LGTM

**Pull request merged**